### PR TITLE
Module 1 - Day 3: Setup and mobile first

### PR DESCRIPTION
**Pull request by lmunozm1702**
The pull request includes:

Repository setup.
Linters configuration.
index.html with navbar and main content for mobile layout.
style.css with design attributes and flex behavior.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Luis Portal</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="toolbar-dark-compact">
+            <div class="logo-placeholder"><img src="assets/images/my_logo.png" alt="Portal logo"></div>
```
Comment (GiuseppeTG) [inline index.html]: - [x] Kindly add an `a` tag to your logo. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <div class="normal-button"><i class="fa-solid fa-bars"></i></div>
```
Comment (GiuseppeTG) [inline index.html]: - [x] Please wrap your hamburger menu with a `nav` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. 🤓

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </header>
+
+        <section class="main-content">
```
Comment (GiuseppeTG) [inline index.html]: - [x] This section should contain the same background image as in the template. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <h1>I’m Mariane Glad to see you!</h1>
```
Comment (GiuseppeTG) [inline index.html]: [ OPTIONAL] You can put your own name here. This goes also for the logo 😸

File: index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            <h5>I’m a software developer! I can help you build a product , feature or website Look through some of my
```
Comment (GiuseppeTG) [inline index.html]: - [x] This paragraph should be a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. 👍

File: index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</h5>
+            <p class="social-media-title">LET'S CONNECT</p>
+
+            <ul class="social-media-icons">
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-linkedin-in"></i></li>
+                <li><i class="fa-brands fa-github"></i></li>
+            </ul>
```
Comment (GiuseppeTG) [inline index.html]: - [x] You are missing some of the social media icons. Please add them and make sure they have an `a` tag.  Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. 🤓

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @lmunozm1702  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
✅ Good commit history and documentation
✅ Nice implementation of flex
✅ Professional PR 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
+.container {
+  max-width: 375px;
+  height: 652px;
+  border: 1px grey solid;
```
Comment (GiuseppeTG) [inline assets/css/style.css]: - [x] Don't forget to take out this border as it is not in the template. I think it was used as a guide, which is ok 😺

File: index.html (hunk 6 above)
Comment (lmunozm1702) [reply index.html]: Hi Guiseppe, please help with this... I don't know the social with the big "M" and the "hand peace" icons. Can you tell me the names of them? please?

File: assets/css/style.css (hunk 7 above)
Comment (lmunozm1702) [reply assets/css/style.css]: It's done, the border was used just to reference the size of the mobile.

File: index.html (hunk 6 above)
Comment (lmunozm1702) [reply index.html]: It's done.

File: index.html (hunk 5 above)
Comment (lmunozm1702) [reply index.html]: It's done.

File: index.html (hunk 4 above)
Comment (lmunozm1702) [reply index.html]: It's done

File: index.html (hunk 3 above)
Comment (lmunozm1702) [reply index.html]: It's done!

File: index.html (hunk 2 above)
Comment (lmunozm1702) [reply index.html]: It's done.

File: index.html (hunk 1 above)
Comment (lmunozm1702) [reply index.html]: It's done

**Review by victorokoroji — APPROVED** [review]
Hi @lmunozm1702, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lmunozm1702** [comment]
Great news for me!!!! thanks a lot!!!

**Pull request merged**